src/dispatching/dialogue/dialogue_dispatcher_handler.rs

```diff
@@ -27,6 +27,6 @@ where
     where
         DialogueWithCx<Upd, D, E>: Send + 'static,
     {
-        Box::pin(async move { self(cx).await })
+        async move { self(cx).await }.boxed()
```

Comment:
Personally, I would go against `.boxed()` in places like this. `Box::pin` is more readable when used with blocks.

Reply:
I mean `f().g().h()` is better, than `f(g(h(...)))`, but at the same time `f(block {})` is better than `block {}.f()` (IMHO)

Reply:
+1

We need to add this rule to our `CODE_STYLE.md` then.